packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -56,6 +56,24 @@ export const isCompiledCSSTemplateLiteral = (
   );
 };
 
+/**
+ * Returns `true` if the expression is using `css` from `@compiled/react` with an object literal.
+ *
+ * @param node {t.Expression} The expression that is being checked
+ * @param meta {Metadata} Useful metadata that can be used during the transformation
+ * @returns {boolean} Whether the node is a css usage from compiled
+ */
+export const isCompiledCSSObjectLiteral = (
```

Comment:
You're not actually checking that the arguments are an object. This should be renamed to `isCompiledCSSCallExpression`, and the function comment should be amended to reflect that.